src/main/resources/migration/common/V60__record_a_recall_calculation_reason.sql

```diff
@@ -0,0 +1,2 @@
+-- These calcs are not pushed to nomis, but nomis_comment populated due to mandatory constraint
```

Comment:
when performing a calc a  reason must be supplied, hence this new reason was inserted